src/components/Sidebar/SidebarHeader/SidebarHeader.tsx

```
@@ -0,0 +1,104 @@
+  };
+
+  return (
+    <div
+      className={`d-flex align-items-center ${hideDrawer ? 'justify-content-center' : 'justify-content-between'}`}
+    >
+      <button
+        className="d-flex align-items-center btn p-0 border-0 bg-transparent"
+        data-testid="toggleBtn"
+        onClick={handleToggle}
+        onKeyDown={handleKeyDown}
+        role="button"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)